### DISCUSSION TITLE
Help regarding submission

### DISCUSSION THREAD

**Discussion by asifrahaman13** [DiveHQ/backend-internship-task#40] [Q&A]
Hi, I have completed the work, but I messed up during my submission. I actually did that in my repo instead of forking it. 

Can anyone tell me what should I do.

The code which I did is as follows:

https://github.com/DiveHQ-Octernships/dive-backend-engineering-octernship-asifrahaman13.git

@OmPrakash95 @prayashm

**Comment by shuklaritvik06** [comment]
> Hi, I have completed the work, but I messed up during my submission. I actually did that in my repo instead of forking it.
> 
> Can anyone tell me what should I do.
> 
> The code which I did is as follows:
> 
> https://github.com/DiveHQ-Octernships/dive-backend-engineering-octernship-asifrahaman13.git
> 
> @OmPrakash95 @prayashm

You have to use the private repository only for the submission, no public PR need to be made for the submission

### Steps to Follow

- [x] Create a new branch `dev` from the main branch
- [x] Commit your changes to the `dev` branch
- [x] Now raise a PR from `dev` to `main`
- [x] Merge that PR
- [x] Your changes would be visible in Feedback#1 PR and there folks will review it
